src/open_inwoner/openklant/tests/data.py

```diff
@@ -26,10 +29,17 @@ def setUpServices(cls):
         )
         config.save()
 
+        oz_config = OpenZaakConfig.get_solo()
+        oz_config.zaak_service = ServiceFactory(
+            api_root=ZAKEN_ROOT, api_type=APITypes.zrc
+        )
+        oz_config.save()
+
     @classmethod
     def setUpOASMocks(self, m):
         mock_service_oas_get(m, KLANTEN_ROOT, "kc")
         mock_service_oas_get(m, CONTACTMOMENTEN_ROOT, "cmc")
+        mock_service_oas_get(m, ZAKEN_ROOT, "zrc")
 
 
 class MockAPIReadPatchData(MockAPIData):
```

Comment:
Let's swap all the `generate_oas_component` for `generate_oas_component_cached` in these MockAPIData classes.